src/app_charts/base/fluent-bit-helm.sh

```diff
@@ -10,6 +10,12 @@ sed -i '1i\{{ if and (eq .Values.robot_authentication "true") (eq .Values.fluent
 sed -i '1i\# !!! DO NOT EDIT THIS FILE !!!\n# This file is autogenerated using src/app_charts/base/fluent-bit-helm.sh.\n# See src/app_charts/base/README.md for update instructions.' ${OUTPUT}
 sed -i '$a\{{ end }}' ${OUTPUT}
 sed -i 's/MY_ROBOT/{{ .Values.robot.name }}/' ${OUTPUT}
+
+# Add a template expression for prepending a subdomain to the fluentbit Tag_Prefix
+# If no subdomain is supplied, the Tag_Prefix will resolve to "kube.var.log.containers."
+# Otherwise, if subdomain is supplied (without the "." at the end), the Tag_Prefix will be "<subdomain>.kube.var.log.containers."
+sed -i 's/Tag_Prefix kube.var.log.containers./Tag_Prefix {{ empty .Values.log_prefix_subdomain | ternary "" (print $.Values.log_prefix_subdomain "." ) -}} kube.var.log.containers./' ${OUTPUT}
```

Comment:
Drop the '$' - this is only needed in a range loop to access the top-level.

Reply:
https://github.com/googlecloudrobotics/core/pull/143/commits/fdf42d5d3a790615e370a076424cad6c23cb8129